src/main/java/com/kolya/gym/controller/TraineeController.java

```diff
@@ -0,0 +1,164 @@
+package com.kolya.gym.controller;
+
+import com.kolya.gym.data.AuthData;
+import com.kolya.gym.data.TraineeData;
+import com.kolya.gym.data.TraineeDataUpdate;
+import com.kolya.gym.domain.Trainee;
+import com.kolya.gym.domain.Trainer;
+import com.kolya.gym.service.TraineeService;
+import com.kolya.gym.service.UserService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@Api(value = "API for trainees", tags = "Trainees")
+@RequestMapping("/trainees")
+@RestController
+public class TraineeController {
+    private final TraineeService traineeService;
+    private final UserService userService;
+
+    private final Logger logger;
+
+
+    @Autowired
+    public TraineeController(TraineeService traineeService, UserService userService, Logger logger) {
+        this.traineeService = traineeService;
```

Comment:
Formatting is scrambled a bit here :)

Reply:
I don't actually understand, where the bad formatting is.